src/models/room.js

```diff
@@ -496,7 +496,7 @@ Room.prototype.loadMembersIfNeeded = function() {
     const inMemoryUpdate = this._loadMembers().then((result) => {
         this.currentState.setOutOfBandMembers(result.memberEvents);
         // now the members are loaded, start to track the e2e devices if needed
-        if (this._client.isRoomEncrypted(this.roomId)) {
+        if (this._client.isCryptoEnabled() && this._client.isRoomEncrypted(this.roomId)) {
```

Comment:
Would it make sense to call `isCryptoEnabled` internally in `isRoomEncrypted`, as the latter is called in a lot more places?

Reply:
Well, I thought about that, but wasn't sure... I mean, you can be in an encrypted room even if your device currently can't load crypto, and `isRoomEncrypted` at least from the method name seems like it should tell you the state of the room in a factual manner without depending on what your device can do.

I agree though that we'd potentially then need to sprinkle `isCryptoEnabled()` to more call sites of `isRoomEncrypted`, but perhaps not all. For example, if there's code that just wants to show an icon on an encrypted room, that's fine with or without crypto on the device. But if you want to trigger key shares or something, such a path really does need to also check `isCryptoEnabled` as well.

What do you think with that additional context?

Reply:
Right, ok, that makes sense. Fine with leaving it as it is then.